content/docs/user-guide/components/reference/terrain/world-debugger.md

```diff
@@ -0,0 +1,25 @@
+---
+linktitle: Terrain World Debugger
+title: Terrain World Debugger Component
+description: The Terrain World Debugger component provides a means to display a wireframe or bounds representaion of the Terrain World.
+toc: true
+---
+
+The **Terrain World Debugger** component allows a representation of the World Bounds to be shown. This visualization can be toggled on or off.
+
+## Provider ##
+
+[Terrain Gem](/docs/user-guide/gems/reference/environment/terrain)
+
+## Dependencies ##
+
+[Terrain World](/docs/user-guide/components/reference/terrain/world)
+
+## Properties ##
+
+![Terrain World Debugger component interface.](/images/user-guide/component/terrain/terrain/ui-terrain-world-debugger-A.png)
```

Comment:
New images should really be created under the proper directory structure for components - i.e., it should match the doc structure. In this case: /images/user-guide/components/reference/terrain. (Also, note that you don't need two "terrain" subdirectories.)

Reply:
Also, the "ui-" prefix in the filename is unnecessary, in my opinion.

Reply:
Changed all three pages to reflect this.